src/uct/cuda/cuda_copy/cuda_copy_iface.c

```diff
@@ -37,14 +37,24 @@ static ucs_config_field_t uct_cuda_copy_iface_config_table[] = {
      "Max number of cuda events. -1 is infinite",
      ucs_offsetof(uct_cuda_copy_iface_config_t, max_cuda_events), UCS_CONFIG_TYPE_UINT},
 
-    {"BW", "10000MBs",
-     "Effective memory bandwidth",
-     ucs_offsetof(uct_cuda_copy_iface_config_t, bandwidth), UCS_CONFIG_TYPE_BW},
+    /* TODO: 1. Add separate keys for shared and dedicated bandwidth
+             2. Remove the "other" key (use pref_loc for managed memory) */
+    {"BW", "h2d:8300MBs,d2h:11660MBs,d2d:320GBs,other:10000MBs",
```

Comment:
Is that possible to remove other and define value for other memory types combinations through value without label?

```suggestion
    {"BW", "10000MBs,h2d:8300MBs,d2h:11660MBs,d2d:320GBs",
```